src/main/scala/com/horizon/exchangeapi/AgbotsRoutes.scala

```diff
@@ -285,6 +285,9 @@ trait AgbotsRoutes extends ScalatraBase with FutureSupport with SwaggerSupport w
     val id = params("id")   // but do not have a hack/fix for the name
     val compositeId = OrgAndId(orgid,id).toString
     authenticate().authorizeTo(TAgbot(compositeId),Access.WRITE)
+    if(!request.body.startsWith("{") && !request.body.endsWith("}")){
```

Comment:
I think you should be tolerant of white space (blanks, tab, newlines) before `{` and after `}`. Same comment goes for all of the other routes.

And also test that that case works.